packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -3866,3 +3866,22 @@ export const getErrorMessageRelatedQuestionUrl = (
               undefined
         : undefined
 }
+
+export function getNextTabParamWhenSwitchingGrapher(
```

Comment:
I struggled to find a place for this function. Let me know if you think this isn't good enough.